domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/Claim.java

```diff
@@ -56,7 +57,8 @@ public Claim(
         final CountyCourtJudgment countyCourtJudgment,
         final LocalDateTime countyCourtJudgmentRequestedAt,
         final Settlement settlement,
-        final LocalDateTime settlementReachedAt
+        final LocalDateTime settlementReachedAt,
+        final String documentSelfPath
```

Comment:
Previous name, something like `sealedClaimDocumentSelfPath`, was better. In this particular context saying what the document is a sealed copy makes sense.

Reply:
done